<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shot Creator</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="job_groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="job_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Job:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="job_comboBox">
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="shotNaming_groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Shot naming</string>
     </property>
     <layout class="QFormLayout" name="formLayout_15">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="prefix_label">
        <property name="text">
         <string>Prefix:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="prefix_comboBox">
        <item>
         <property name="text">
          <string>SH</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PC</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="start_label">
        <property name="text">
         <string>Numbering start:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="start_spinBox">
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="suffix_label">
        <property name="text">
         <string>Suffix:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="suffix_lineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="createSeq_checkBox">
        <property name="text">
         <string>Create sequence</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="shotCount_spinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="shotCount_label">
        <property name="text">
         <string>Number of shots:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="increment_spinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="increment_label">
        <property name="text">
         <string>Increment:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="shotPreview_groupBox">
     <property name="title">
      <string>Preview</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPlainTextEdit" name="shotPreview_plainTextEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="shotCreator_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>prefix_comboBox</tabstop>
  <tabstop>createSeq_checkBox</tabstop>
  <tabstop>start_spinBox</tabstop>
  <tabstop>suffix_lineEdit</tabstop>
  <tabstop>shotPreview_plainTextEdit</tabstop>
  <tabstop>shotCreator_buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../../ui/rsc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>shotCreator_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shotCreator_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createSeq_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>shotCount_spinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createSeq_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>shotCount_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>51</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createSeq_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>increment_spinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createSeq_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>increment_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createSeq_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>start_spinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createSeq_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>start_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>41</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
